specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/CommonDefinitions.json

```diff
@@ -3332,7 +3332,8 @@
             "LeastResponseTime",
             "WeightedTotalTraffic",
             "RequestHash",
-            "PerSiteRoundRobin"
+            "PerSiteRoundRobin",
+            "LeastRequestsWithTieBreaker"
```

Comment:
why not 2023-12-01? If you dont move this there, could you include it in that one as well?